firefox_ui_tests/remote/security/test_mixed_script_content_blocking.py

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
For a final review please do not include this file. Thanks. Lets see if we can get the other PR landed soon.